spec/unit/dao/cassandra/factory_spec.lua

```diff
@@ -23,6 +23,7 @@ describe("Cassadra factory", function()
         contact_points = dao_properties.contact_points,
         keyspace = dao_properties.keyspace,
         query_options = {
+          consistency = 1,
```

Comment:
this should use the value from the `cassandra.consistencies` enum and not the underlying value